package.json

```diff
@@ -45,6 +45,7 @@
   "dependencies": {
     "commander": "^7.2.0",
     "handlebars": "^4.7.7",
+    "import-cwd": "^3.0.0",
```

Comment:
we should not depend on packages from this author; they've committed to making them all ESM-only.

Reply:
eh... I don't really want to weigh in on a "don't use packages from sindresorhus" conversation regardless of what I personally think about the topic but this isn't an issue for `import-cwd` anyway since it's not possible to actually "import from cwd" in ESM land 🤔

Reply:
it definitely is, with dynamic import.

Reply:
so... I don't understand what is happening here. I am submitting a feature in good faith (based on a suggestion in my previous PR review) and suddenly things have gotten incredibly hostile. And I don't understand why you're brining this up @ljharb, are you a maintainer of this project or just a "concerned bystander"?

And on the point of it not being possible to "import from cwd" I get the impression that we're talking past each other here. I was implicitly saying that you can't "import from cwd" without changing the structure of the code significantly. As it turns out we're doing a dynamic require here anyway and already in an async function so it's possible to do import with a dynamic import here but that's not the point I was trying to make. If you want some more context you can follow the thread through the following links: 

https://github.com/sindresorhus/import-cwd/issues/3
https://github.com/sindresorhus/import-from/issues/9
https://github.com/nodejs/node/issues/36396
https://github.com/nodejs/loaders

Reply:
Additionally, I’m quite familiar with loader hooks and the many blockers to ESM being as usable as CJS; the issue is that most packages will never migrate to CJS and should not, and this, deps committing to going ESM-only are inevitably going to be unusable for most.

Reply:
I’m a bit out of touch with this stuff, sorry guys. What’s the worst that can happen here? We get stuck using `v3` of this library and can’t upgrade?

Are there any safer alternatives?

Reply:
Yes, that's right. The safer alternative i to either use a different author's package, or to write it ourselves with `fs.readdir`.

Reply:
@cookpete yes that's my perspective on this, it won't start failing overnight unless we update a major version of the dependency. My personal view is that this library is working as-is and we shouldn't "prematurely optimise" our approach based on a hypothetical future 🤷 

But I'm more interested in getting this PR merged than having any philosophical or political debates so if you say that you would prefer me to remove this library then I'll do it 👍

Reply:
We're still not safe; he plans to migrate all of them eventually, but it's fine to land it as long as we're aware that we'll want to migrate off it ASAP.